chatmaild/src/chatmaild/ini/chatmail.ini.f

```diff
@@ -33,7 +33,7 @@
 passthrough_senders =
 
 # list of e-mail recipients for which to accept outbound un-encrypted mails
-passthrough_recipients =
+passthrough_recipients = xstore@testrun.org groupsbot@hispanilandia.net
```

Comment:
what is groupsbot@hispanilandia.net needed for if you can't write in mailing lists? to subscribe to channels?

btw, maybe process the json file at https://github.com/deltachat-bot/public-bots/blob/main/frontend/data.json